src/core/client/stream/tabs/Comments/Stream/PostCommentForm/PostCommentFormContainer.tsx

```diff
@@ -117,14 +117,21 @@ export const PostCommentFormContainer: FunctionComponent<Props> = ({
 
   const handleOnSubmit: OnSubmitHandler = async (input, form) => {
     try {
-      const response = await createComment({
+      const params: any = {
         storyID: story.id,
         nudge,
         commentsOrderBy,
         body: input.body,
         rating: input.rating,
         media: input.media,
-      });
+      };
+
+      if (tab === "UNANSWERED_COMMENTS") {
+        params.tag = GQLTAG.UNANSWERED;
+        params.connectionKey = "UnansweredStream_comments";
+      }
```

Comment:
This is incomplete, and would only work for this particular tab.
Look at how R&R is determining its tag inside the `CreateCommentMutation`. I think we should follow the same patern.